SUSE/scf
# Added tooling to assess minibroker charts for usability with CAP

## Description

https://jira.suse.com/browse/CAP-566

Main script `assess-minibroker-charts` to perform assessment based on
current state of master index of all stable helm chart.

Secondary script to convert base results into the yaml format expected
by other tooling.

Helper scripts for various sub-tasks.

Patch files required by charts to work with CAP/minibroker.

Documentation (Requisites, usage).

## Test plan

  - Create CAP cluster, point `kubectl` at it.
  - Run main script as per its documentation. 
  - Wait two days (for completion), or just long enough to see a single working chart.
  - Generate yaml from main results, using the secondary script, as per its documentation.
  - :warning: The referenced ticket has a `dbe-chart-results.csv` attached, my results. This enables testing in/of incremental mode as well, which is much faster. Just removing specific entries in the file can be used to trigger testing of known (non-)working charts.

:warning: At this point the tooling is not integrated into any type of pipeline.
